lintro/ai/review/path_utils.py

```
@@ -0,0 +1,100 @@
+        f"{source_stem}.test.js",
+        f"{source_stem}.spec.jsx",
+        f"{source_stem}.test.jsx",
+        f"test_{source_stem}.tsx",
+        f"test_{source_stem}.jsx",
+    }
+    if name not in expected_names:
+        return False
+
+    if name.startswith(f"test_{source_stem}.") or name.startswith(
+        f"{source_stem}_test.",
+    ):
```

Ruff (0.15.18)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)